files/class-vip-filesystem-stream-wrapper.php

```diff
@@ -314,16 +316,14 @@ public function stream_flush() {
 
 		try {
 			// Upload to file service
-			$result = $this->client
-				->upload_file( $this->uri, $this->path );
+			$result                   = $this->client->upload_file( $this->uri, $this->path );
+			$this->should_flush_empty = false;
```

Comment:
This avoids the "double flush": if we run `flush()` before we close the file, we don't want to upload the file twice (see [this line](https://github.com/Automattic/vip-go-mu-plugins/pull/3853/files#diff-9ff26734fc2ae14c41038dfa03c7c03f156e31d68a8ffac8d379f808ad9f20c5L243)).